test/e2e_node/summary_test.go

```diff
@@ -35,7 +35,7 @@ import (
 	"github.com/onsi/gomega/types"
 )
 
-const restartCount = 3
+const restartCount = 1
```

Comment:
yes, sorry.  Confusing naming.  This doesnt refer to that restartCount.  This is just a constant for the test that tells the pod how many times to restart.  It does this by writing to a file in a volume, and determining whether or not to restart based on what it reads from the volume.  So this will make it exit once, and not do anything the next time.